packages/types/src/srml/parachains/types.ts

```diff
@@ -0,0 +1,167 @@
+// Auto-generated, do not edit
```

Comment:
At least this gets sorted.

Reply:
Why is this file committed?

The workflow I was thinking about was:
- the user runs smrlTypes.ts script, it outputs this file somehwere
- the user then needs to manually add this file to tsconfig's typeRoot

(and have a look if 0x and typechain does something more intelligent too)

Reply:
Agreed with your workflow.

However, would at least like to provide the default types for polkadot longer term. (Actually happy to add the other chain defaults as well, as long as I don't need to generate them)

As of now, don't want to boil the water too much - so there are still imports, just not classes. (At least for some)